docs/scratch_orgs.rst

```diff
@@ -1,38 +1,201 @@
-Manage scratch orgs
+Manage Scratch Orgs
 ===================
 
-What is an org?
----------------
+*Scratch orgs* are ephemeral Salesforce orgs. Scratch orgs are created empty, unlike sandboxes, and configured by deploying metadata into them. Scratch orgs facilitate repeatable, fully source-driven development processes on the Salesforce platform, and help ameliorate many of the challenges inherent in working with *persistent orgs* like sandboxes and Developer Edition orgs. Scratch orgs are intended to be created and destroyed often; they cannot have a lifespan greater than 30 days.
+
+The CumulusCI Suite offers tools for working with all types of Salesforce orgs, but provides the most value when working with scratch orgs. We strongly encourage building processes around scratch orgs for development, feature and integration testing, demos, and other workflows. CumulusCI automation helps realize the promise of scratch orgs as low cost, repeatable, source-driven environments for every phase of the product lifecycle.
```

Comment:
My take on the above paragraphs:

Scratch orgs are temporary Salesforce orgs which can be quickly set up "from scratch" and which last for a limited time (no more than 30 days). We strongly encourage using scratch orgs for development and QA, instead of sandboxes or Developer Edition orgs, because:

- They provide a repeatable starting point without the cruft that can accumulate in persistent orgs over time.
- The limited lifespan motivates making sure that all steps to set up the org are performed by CumulusCI automation with configuration stored as part of the project's source repository, which makes it easier to repeat the same steps in other orgs.